griptape/schemas/union_field.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import annotations
+
+from typing import Any
+
+import marshmallow
+import marshmallow.error_store
+import marshmallow.exceptions
+
+
+class MarshmallowUnionError(Exception):
+    """Base error for marshmallow_union."""
+
+
+class ExceptionGroupError(MarshmallowUnionError):
+    """Collection of possibly multiple exceptions."""
+
+    def __init__(self, msg: str, errors: Any) -> None:
+        self.msg = msg
+        self.errors = errors
+        super().__init__(msg, errors)
+
+
+class UnionField(marshmallow.fields.Field):
+    """Field that accepts any one of multiple fields.
```

Comment:
Please include a link to the source repo https://github.com/adamboche/python-marshmallow-union in the description.

Reply:
@william-price01 please still add this, even with the `NOTICE`

Reply:
Weird, I added this, can you not see the changes?